group_vars/all.yml.dist

```diff
@@ -59,6 +60,37 @@ ansible_nas_user: david
 # Your email address, used for LetsEncrypt SSL certs
 ansible_nas_email: me@email.com
 
+###
+### Watchtower
+###
+# Sets the 6 field cron schedule to use for checks and updates
+# Keep in mind that this will stop and start containers if there is an update
+# So probably best not to do it every 10 minutes unless you want your plex stream
+# interrupted
+# Default below is every day at 5am (local time if you have set
+# your local timezone above)
+watchtower_cron_schedule: "0 0 5 * * *"
+
+## Notifications
+# If enabling notifications, you MUST correctly setup the relevant
+# section below
+watchtower_notifications_enabled: false
+# Set the type of notification. email or slack
+watchtower_notifications_type: email
+# Set the notifications level. panic, fatal, error, warn, info (default) or debug
+watchtower_notifications_level: info
+
+# Email
+watchtower_email_from: "{{ ansible_nas_email }}"
+watchtower_email_to: "{{ ansible_nas_email }}"
+watchtower_email_server: smtp.gmail.com
```

Comment:
I think it makes sense to have an outbound email server/username/password setting bit in the general section, it shouldn't be specific to watchtower, and perhaps suggest mailgun/sendgrid/whatever

Reply:
That makes sense. I'll change it tonight